index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MyPortofolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+
+</head>
+
+<body class="body">
+
+    <header>
+
+
+        <nav class="navbar fixed-top">
+            <a href="" id="logo">
+                Welcome
+              </a>
+
+            <img src="/Images/Union.png" alt="navigation bars" class="menu">
+
+        </nav>
+
+    </header>
+
+
+    <section class="headline">
+        <div class="container">
+            <h1 class="identity">
+                Hey there. I'm, <br/> Tite Lunda <br><span>I'm a software developer</span>
+            </h1>
+            <p class="profile">
+                <b></b> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me. </p>
+
+            <ul class="icon-group">
+                <li class="icon"> <a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                <li class="icon"> <a href=""><i class="fa-brands fa-linkedin"></i></a></li>
+                <li class="icon"> <a href=""><i class="fa-solid fa-m"></i></a></li>
+                <li class="icon"> <a href="#https://github.com/TiteLunda"><i class="fa-brands fa-github"></i></a></li>
+                <li class="icon"> <a href=""><i class="fa-brands fa-angellist"></i></a></li>
+            </ul>
+
+            <div class="chevron">
+                <span class="fa-solid fa-chevron-down"></span>
+            </div>
+        </div>
+
+    </section>
+    <section class="works ">
+        <div class="TitleWorks ">
+            <h2 class="Recent ">My Recents Works</h2>
+            <hr class="line ">
+        </div>
+
+        <div class="cards ">
+            <div class="cardsContent ">
+                <img src="./Images/Img Plaholder.png " alt="Store image ">
+                <h3 class="title1 ">Multi-Post Stories Gain+Glory</h3>
+                <ul class="Languages ">
+                    <li class="language ">Ruby on rails </li>
+                    <li class="language ">Css </li>
```

Comment:
- [ ] Kindly, make sure you fixed your `margin` to stick to the design as much as possible using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=7V3y8z3F5uwwsjzJ-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Capture](https://user-images.githubusercontent.com/57408419/228301352-8f70498a-0f4a-4f67-a2ca-5bffc424fe5e.JPG)| |![Capture0](https://user-images.githubusercontent.com/57408419/228301518-6640c349-90c4-4859-947b-19340d5cf503.JPG)| |